### PR TITLE
Added sns to sentinel-2

### PR DESCRIPTION
**Pull request by carioca-au**

### CODE REVIEW

**Review by alexgleith — APPROVED** [review]

File: data_specs/Sentinel-2_Level-2A_specs.ipynb [hunk 1]
```diff
@@ -355,7 +355,48 @@
...
    "source": [
-    "**Table 5: AWS file path convention**\n",
+    "### Amazon Simple Notification Service "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Sentinel-2 Level-2A stacs can be received thanks to the Public Dataset Program."
```
Comment (alexgleith) [inline data_specs/Sentinel-2_Level-2A_specs.ipynb:365]:
```suggestion
    "Sentinel-2 Level-2A STAC documents can be received via a SNS notification"
```

File: data_specs/Sentinel-2_Level-2A_specs.ipynb [hunk 2]
```diff
@@ -355,7 +355,48 @@
...
+    "| Region |`af-south-1` |\n",
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The SNS endpoints are in the AWS region `af-south-1` (Cape Town). Additional region specifications can be applied as follows:\n",
+    "\n",
+    "`aws sns list-topics --region=af-south-1 `\n",
```
Comment (alexgleith) [inline data_specs/Sentinel-2_Level-2A_specs.ipynb:391]: You really can't list these public topics like this... this is not helpful.

**Review by alexgleith — CHANGES_REQUESTED** [review]
Think about what you're writing! You can't list SNS topics from someone else's account like that.

File: data_specs/Sentinel-2_Level-2A_specs.ipynb (hunk 2 above)
Comment (carioca-au) [reply data_specs/Sentinel-2_Level-2A_specs.ipynb:391]: removed

**Comment by carioca-au** [comment]
> Think about what you're writing! You can't list SNS topics from someone else's account like that.

Yeah, I haven't thought about that. 
Removed

**Pull request closed without merging**